packages/protocol/contracts/dapis/SamplePriceDataFeed.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
```

Comment:
ezpz lemon squeezy data feed

Reply:
yea, i like this. it makes sense to have the requester reset storage, cuz they know when requests get made/fulfilled

Reply:
what happens if `makeDapiRequest` is called again before the last request gets processed? it just gets "overwritten", and that doesn't matter since the price feed is concerned with the most up-to-date responses?

Reply:
The Airnode fulfillments to the previous request will get reverted, meaning that we will not receive a reduce callback from them.
> it just gets "overwritten", and that doesn't matter since the price feed is concerned with the most up-to-date responses?

Yes